iscsi/initiator.go

```diff
@@ -23,6 +24,10 @@ var (
 
 const (
 	iscsiBinary = "iscsiadm"
+	manualMode = "manual"
```

Comment:
`scanModeManual`, `scanModeAuto`, `scanTimeout`

We can separate the two timeouts later.